dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -0,0 +1,310 @@
+<!-- This is not the final file that gets used, this is processed by the SbtMavenPlugin, in the sbt project directory. -->
```

Comment:
@dotta Checkout the new plugin.xml, not only are properties that can be determined from the build generated, but I've created a parameterManagement section that saves on the duplication of parameter definitions for each mojo.